roles/pulp_content_guard_rbac/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- name: Get RBAC content guard list
+  uri:
```

Comment:
It's a shame there's no module for this in squeezer.

Reply:
Matt and I discussed writing one rather than adding it directly into our collection but it would probably take a while to get a PR approved.